lms/templates/problem_notifications.html

```diff
@@ -0,0 +1,22 @@
+<%page expression_filter="h" args="notification_type, notification_icon, notification_message, hint_notification"/>
+<%!
+from django.utils.translation import ugettext as _
+from openedx.core.djangolib.markup import HTML
+%>
+<%
+    notification_type = "info"
+    notification_icon = "question"
+    notification_message = "Hint (1 of 2): A fruit is the fertilized ovary from a flower. Hint (2 of 2): A fruit contains seeds of the plant."
+    hint_notification = True
+%>
+
+<div class="notification ${notification_type}">
```

Comment:
Are we moving focus here, or relying on aria-live? If we're moving focus, we will need a tabindex=-1 and if its the latter we will need an aria-live="polite"